scripts/gen_coverage_report.sh

```diff
@@ -30,7 +30,7 @@ function cov {
 cargo llvm-cov --workspace clean
 cargo llvm-cov --workspace --no-report
 cov forest-cli --chain calibnet db clean --force
-cov forest-cli --chain calibnet snapshot fetch --aria2 --provider filecoin --compressed -s "$TMP_DIR"
+cov forest-cli --chain calibnet snapshot fetch --aria2 --provider filecoin -s "$TMP_DIR"
```

Comment:
Are there any subcommands that you introduced in this PR but didn't cover in Rust tests?